suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -1,50 +1,76 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 
-import { selectCoins } from '@suite-common/wallet-core';
-import { useFormatters } from '@suite-common/formatters';
-import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { pipe } from '@mobily/ts-belt';
+
+import { convertCryptoToFiatAmount, useFormatters } from '@suite-common/formatters';
+import { FiatRates, WalletAccountTransaction } from '@suite-common/wallet-types';
 import { Card, Table, Td, Text, Th, Tr, VStack } from '@suite-native/atoms';
-import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
-import { FiatCurrency } from '@suite-common/suite-config';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { selectCoins } from '@suite-common/wallet-core';
+import { selectFiatCurrency } from '@suite-native/module-settings';
+import { CryptoToFiatAmountFormatter } from '@suite-native/formatters';
 
 import { TransactionDetailSheet } from './TransactionDetailSheet';
 
 type TransactionDetailValuesSheetProps = {
     isVisible: boolean;
     transaction: WalletAccountTransaction;
-    fiatCurrency: FiatCurrency;
     onSheetVisibilityChange: () => void;
 };
 
 type TodayHeaderCellProps = {
-    historicalPrice: string | number;
-    actualPrice: string | number;
+    historicalPrice: string;
+    actualPrice: string;
+    network: NetworkSymbol;
+    historicalRates?: FiatRates;
 };
 
-const TodayHeaderCell = ({ historicalPrice, actualPrice }: TodayHeaderCellProps) => {
-    const fiatTotalHistoryNumeric = Number(historicalPrice);
-    const fiatTotalActualNumeric = Number(actualPrice);
+const TodayHeaderCell = ({
+    historicalPrice,
+    actualPrice,
+    network,
+    historicalRates,
+}: TodayHeaderCellProps) => {
+    const { SignValueFormatter } = useFormatters();
+    const coins = useSelector(selectCoins);
+    const fiatCurrency = useSelector(selectFiatCurrency);
+
+    const fiatTotalHistoryNumeric = pipe(
```

Comment:
Seems to me you are kind of overusing `pipe` for such a simple cases where is very small chance it will need to be extended in future. But you can keep this if you like it :D

Reply:
Yeah a little bit. I'm curious whether it's really going to make it easier to read this code when we use it everywhere like you suggested when we introduced ts-mobily. I'll keep it and avoid simple cases like this in the future.

Reply:
Yea, but as I said I kind of like it anyway, just be careful with that.